chanacceptor/rpcacceptor.go

```diff
@@ -331,6 +331,13 @@ func (r *RPCAcceptor) sendAcceptRequests(errChan chan error,
 				):
 					commitmentType = lnrpc.CommitmentType_ANCHORS
 
+				case channelFeatures.OnlyContains(
```

Comment:
This needs to be in the funding manager as well, as otherwise it'll reject the request (that's the error message you see, notice how here it just logs when it's unknown). 

Also I think you want just alias required and static key, iiuc, CLN doesn't send the zero conf bit and relies on extra-protocol negotiation to have the other side send min depth zero early. We also still don't want to support the zero conf bit w/ static key, but alias with static key is fine.

Reply:
Also an itest should be added to make sure the desired scenario works.

Reply:
@roasbeef ok started working on this. Is the spot you are referencing for funding manager?
https://github.com/lightningnetwork/lnd/blob/master/funding/commitment_type_negotiation.go#L65

I started to add it in there, but I noticed there is no `CommitmentTypeStaticRemoteKey` like I was expecting to find for the commitment type. Is `CommitmentTypeLegacy` what I want there instead?
https://github.com/lightningnetwork/lnd/blob/master/lnwallet/reservation.go#L71-L85

Reply:
tweakless is what you want